checks/mongo-check_test.go

```diff
@@ -0,0 +1,43 @@
+package checks
+
+import (
+	"fmt"
+	"testing"
+
+	"go.mongodb.org/mongo-driver/mongo/integration/mtest"
+)
+
+func TestMongoCheck_Pass(t *testing.T) {
+	mt := mtest.New(t, mtest.NewOptions().CreateClient(true).ClientType(mtest.Mock))
+	defer mt.Close()
+
+	fmt.Println(mt.Client)
+	check := NewMongoCheck(2, mt.Client)
+	if !check.Pass() {
+		t.Errorf("Expected MongoCheck.Pass to return true, got false")
+	}
+}
+
+func TestMongoCheck_Name(t *testing.T) {
+	check := NewMongoCheck(2, nil)
+	if check.Name() != "mongodb" {
+		t.Errorf("Expected MongoCheck.Name to return 'mysql', got '%s'", check.Name())
```

Comment:
I think there is a typo in error message (it should return `mongodb`, not `mysql`)